### PR TITLE
Bump to latest parent pom

### PR DESCRIPTION
**Pull request by batmat**
@francisu @ndeloof as the current people declared in the permission file as releasers.

cc @reviewbybees for more eyes.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -124,12 +125,12 @@ THE SOFTWARE.
         <dependency>
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpclient</artifactId>
-            <version>4.5.2</version>
+            <version>4.5.3</version>
```
Comment (dwnusbaum) [inline pom.xml]: Better to use the [apache-httpcomponents-client-4-api](https://github.com/jenkinsci/apache-httpcomponents-client-4-api-plugin) plugin instead if possible.

Comment (batmat) [reply pom.xml]: @dwnusbaum agreed, but here my only goal was to get past the RequireUpperBounds failure. IMO what you're saying is definitely the way to go, but should be a dedicated ticket.

Comment (dwnusbaum) [reply pom.xml]: @batmat Agreed, I think it makes sense to keep this PR minimal and handle migrating to the API plugin separately.

**Comment by batmat** [comment]
@francisu any possibility of a merge & release?

If you do not have time anymore for this plugin, that is fine too. Just please answer here with a comment, so that we can mark the plugin to be adopted.

Thanks a lot!

File: pom.xml (hunk 1 above)
Comment (oleg-nenashev) [reply pom.xml]: Agreed

File: pom.xml [hunk 2]
```diff
@@ -207,14 +208,6 @@ THE SOFTWARE.
...
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-compiler-plugin</artifactId>
-                <configuration>
-                    <source>1.7</source>
-                    <target>1.7</target>
-                </configuration>
-            </plugin>
             <plugin>
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>findbugs-maven-plugin</artifactId>
```
Comment (oleg-nenashev) [inline pom.xml]: can be replaced by a `findbugs.failOnError` property so that there are reports during the build

**Comment by batmat** [comment]
@francisu you've done an awesome work for a long time here. The community is very grateful for your efforts! 
If you cannot anymore work on this, that is fine. This is life and happens to the bests :-). Maybe time will come again :-). 
I am going to mark the plugin as up for adoption then. Thank you again

**Comment by ndeloof** [comment]
not sure why I'm set as a releaser. Definitively not maintaining this plugin

**Pull request merged**